run-cnf-suites.sh

```diff
@@ -79,6 +80,9 @@ done
 # Strips the leading whitespace.
 LABEL="$(echo -e "${LABEL}" | sed -e 's/^[[:space:]]*//')"
 
+if [[ $LABEL == "all" ]]; then
+	LABEL='common,extended,faredge,telco'
+fi
 # List the specs (filtering by suite).
```

Comment:
```suggestion
[ "$LABEL" = all ] && LABEL=common,extended,faredge,telco

# List the specs (filtering by suite).
```

Reply:
I think the way it is has a good amount of readability.  We can keep it this way.